birdhouse/components/stac/populator/populate.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
I'm not quite sure if there is any benefice of using this separate bash script over `argparse` and `subprocess`/`multiprocess`/`threading` calls in the same Python file that could combine `stac_generator.scripts.stac_generator` and `collection_processorCollectionProcessor`.

It seems working all under Python would reduce the number of input validations, would simplify chaining them and makes it easier to extend with more arguments or functionalities.